definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -216,13 +216,6 @@
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "R"
   },
-  {
-    "LiveFrom": "16/09/2020",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "solAosNotReceivedFromServiceAppRejected",
```

Comment:
Should we wait until Deemed and dispensed is released to do that?

Reply:
I'm trying to figure out with Jeremy if this should go to prod now and be removed on release of RRJ or already rtemoved from prod

Reply:
Question to answer: Should we leave CRU for release (DnD) on Monday and this will be changed when RRJ is released?

OR we want to already change it for DnD release?

Reply:
My opinion is that we should wait for it to be released. Deliver what we said we would in the past and then come with this AC after Mondays release of D&D.